Project.toml

```diff
@@ -1,10 +1,12 @@
 name = "Lighthouse"
 uuid = "ac2c24cd-07f0-4848-96b2-1b82c3ea0e59"
 authors = ["Beacon Biosignals, Inc."]
-version = "0.13.4"
+version = "0.14.0"
```

Comment:
Bumping the version b/c we've changed the expected output for "failure" cases from `missing` to `NaN` for any metrics output types that show up in vectors (to aid in Arrow serialization).